data/json/recipes/weapon/ranged.json

```
@@ -1206,5 +1206,20 @@
     "autolearn": true,
     "qualities": [ { "id": "SAW_M_FINE", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
     "components": [ [ [ "nailgun", 1 ] ], [ [ "pipe", 1 ] ], [ [ "2x4", 2 ], [ "stick", 3 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "coilgun",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "electronics",
+    "skills_required": [ [ "mechanics", 5 ], [ "electronics", 4 ] ],
+    "difficulty": 4,
+    "time": "25 m",
```

```suggestion
    "time": "1 h",
```
Such a complex recipe should at least use 1 hour